assets/section-multicolumn.css

```diff
@@ -88,6 +88,16 @@
   margin-top: 1rem;
 }
 
+.multicolumn-card__info a:after {
+  bottom: 0;
+  content: "";
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  z-index: 1;
+}
```

Comment:
I don't think it's ideal to make the whole thing a link 🤔 
Doing so makes it that you can't select the text content and copy it.

On top of that you can use the column rich text setting to add some links into the text which become un-clickable because we're now covering the whole card with our link label  

I think the clicking area is decent enough the way we have it so I'm not sure it needs changing. 

[video](https://screenshot.click/14-45-5q45g-p30wc.mp4)